apps/dashboard/src/locales/en/modules/seller.json

```diff
@@ -57,6 +57,36 @@
           "selectCategory": "Select category",
           "selectVat": "Select VAT"
         }
+      },
+      "payouts": {
```

Comment:
These do not go into the sellers module, but into the financial module.

Reply:
seller payouts are all under the seller module. When we have RBAC access I will rewrite it such that users can _view_ their payouts.